draft-ietf-quic-ack-frequency.md

```diff
@@ -163,17 +163,15 @@ endpoint performance in the following ways:
   opportunities available to other flows sharing the same link.
 
 
-As discussed in {{implementation}} however, there can be undesirable consequences
-to congestion control and loss recovery if a receiver unilaterally reduces the
-acknowledgment frequency. A sender's constraints on the acknowledgment
-frequency need to be taken into account to maximize congestion controller and
-loss recovery performance.
+However, as discussed in {{implementation}}, a unilateral reduction in
+acknowledgement frequency can lead to undesirable consequences for congestion
+control and loss recovery.
 
-{{QUIC-TRANSPORT}} specifies a simple delayed acknowledgment mechanism that a
-receiver can use: send an acknowledgment for every other packet, and for every
-packet that is received out of order ({{Section 13.2.1 of QUIC-TRANSPORT}}).
-This does not allow a sender to signal its preferences or constraints. This
-extension provides a mechanism to solve this problem.
+{{QUIC-TRANSPORT}} specifies a simple delayed acknowledgment mechanism
+({{Section 13.2.1 of QUIC-TRANSPORT}}).  A data sender's constraints on the
```

Comment:
```suggestion
({{Section 13.2.1 of QUIC-TRANSPORT}}) without any ability for the data
sender to impact this behavior.  A data sender's constraints on the
```
I feel this part is missing from the flow as you removed the middle sentence ("This does not allow..."). Note I copied this wording from the intro text now.